scene/resources/3d/importer_mesh.cpp

```diff
@@ -421,6 +421,9 @@ void ImporterMesh::generate_lods(float p_normal_merge_angle, Array p_bone_transf
 			}
 		}
 
+		if (index_count >= indices.size()) {
+			throw std::runtime_error("Bar array access impending.");
```

Comment:
We don't usually use the STL in the project.

https://docs.godotengine.org/en/stable/contributing/development/cpp_usage_guidelines.html#standard-template-library